sdk/automation/Azure.ResourceManager.Automation/tests/AutomationManagementTestBase.cs

```diff
@@ -13,6 +13,9 @@ namespace Azure.ResourceManager.Automation.Tests
     public class AutomationManagementTestBase : ManagementRecordedTestBase<AutomationManagementTestEnvironment>
     {
         protected ArmClient Client { get; private set; }
+        protected AzureLocation DefaultLocation => AzureLocation.EastUS;
```

Comment:
The default location should be `TestEnvironment.Location`

Reply:
fixed